addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,796 @@
-                            </small>
-                            <h4>
-                                <a itemprop="url" t-att-class="event.state == 'done' and 'text-success'" t-attf-href="/event/#{ slug(event) }/#{(not event.menu_id) and 'register' or ''}"><b itemprop="name" t-field="event.name"/></a>
-                            </h4>
-                            <small t-if="event.organizer_id" class="text-muted">Organized by <span t-field="event.organizer_id" /></small>
-                            <div>
-                                <i class="fa fa-clock-o text-primary mx-1" role="img" aria-label="Start date" title="Start date"/>
-                                <span itemprop="startDate" t-field="event.with_context(tz=event.date_tz).date_begin" t-options='{"hide_seconds":"True"}'></span>
-                                <small>to</small>
-                                <span itemprop="endDate" t-field="event.with_context(tz=event.date_tz).date_end" t-options='{"hide_seconds":"True"}'></span>
+        <div id="wrap" class="o_we_index">
+            <!-- Options -->
+            <t t-set="opt_events_list_cards" t-value="request.website.viewref('website_event.opt_events_list_cards').active"/>
+            <t t-set="opt_events_list_columns" t-value="request.website.viewref('website_event.opt_events_list_columns').active"/>
+            <!-- Topbar -->
+            <t t-call="website_event.index_topbar"/>
+            <!-- Drag/Drop Area -->
+            <div id="oe_structure_we_index_1" class="oe_structure"/>
+            <!-- Content -->
+            <main t-attf-class="o_we_events_list #{'opt_event_list_cards_bg' if opt_events_list_cards else ''}">
+                <div class="container">
+                    <div class="row">
+                        <div id="o_we_index_main_col" t-attf-class="col-md my-5 #{'opt_events_list_columns' if opt_events_list_columns else 'opt_events_list_rows'}">
+                            <div class="row">
+                                <!-- Events List -->
+                                <t t-call="website_event.events_list"/>
                             </div>
-                            <hr/>
-                            <t t-if="not event.is_online">
-                                <div itemprop="location" t-field="event.address_id" t-options='{
-                                    "widget": "contact",
-                                    "fields": ["city"]
-                                }'/>
-                            </t>
                         </div>
                     </div>
                 </div>
-            </div>
-            <div class="form-inline justify-content-center my-3">
-                <t t-call="website.pager"/>
-            </div>
+            </main>
+            <!-- Drag/Drop Area -->
+            <div id="oe_structure_we_index_2" class="oe_structure"/>
```

Comment:
SRI developed a thing to display a message in the drop area which can indicate to the user this is an area which is shown in all events (I don't know if this is the case for this particular one but you might want to use this somewhere in your task). It is called data-editor-sub-message or something like that ;)